tests/test_core/conftest.py

```diff
@@ -75,15 +16,66 @@ async def p9(request, tmpdir):
     await ctx.close()
 
 
+@pytest.fixture
+def nsname():
+    ns = uifname()
+    netns.create(ns)
+    with IPRoute(netns=ns) as ipr:
+        ipr.link('set', index=1, state='up')
+        ipr.poll(ipr.addr, 'dump', address='127.0.0.1', timeout=5)
+    yield ns
+    try:
+        netns.delete(ns)
+    except Exception:
+        pass
+
+
+@pytest.fixture
+def link(nsname):
```

Comment:
I  think fixture names should be a bit more explicit, so less guessing when trying to figure later what tests do